### PR TITLE
README：Soporte español

### PR DESCRIPTION
**Pull request by itsFrankV22**
I am willing to help with support and translation into Spanish, I verify and approve that the translation is as correct as possible

Doc: Added README_ES.md
Doc: Update README_EN.md and README.md to support README_ES.md

### CODE REVIEW

**Comment by itsFrankV22** [comment]
西班牙语支持

File: README_ES.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+## Notas de usuario
+
+- Tenga en cuenta que algunos complementos pueden requerir dependencias; consulte la lista a continuación para instalar las dependencias.
+- Cada complemento tiene una nota de uso; haga clic en el hipervínculo en la lista a continuación para ver las instrucciones específicas.
+- Se dice que a las personas a las que les gusta destacar repositorios, sus complementos no son fáciles de generar errores.
+
+## Descargas
+
+- Github Release: [Plugins.zip](https://github.com/UnrealMultiple/TShockPlugin/releases/download/V1.0.0.0/Plugins.zip)
+- Gitee Release: [Plugins.zip](https://gitee.com/kksjsj/TShockPlugin/releases/download/V1.0.0.0/Plugins.zip)
```
Comment (sgkoishi) [inline README_ES.md:31]: it's quite unlikely that a Spanish user use Gitee

File: README_ES.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+#### AutoPluginManager
+    /apm l Lista todos los plugins
+    /apm i <plugin name> Una vez - Instalar Plugin
+    /apm u [plugin name] Ver si el plugin tiene actualizacion
+
+## Notas de Desarrollador
+
+> Estandarización de codigo
+
+- No utilice nombres de variables chinos.
```
Comment (sgkoishi) [inline README_ES.md:42]: or use Chinese variables

**Comment by Controllerdestiny** [comment]
pr你可以使用用英语标题

File: README_ES.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+> [!NOTE]
+> Es posible que la documentación del complemento en inglés no se actualice tan rápidamente como la documentación del complemento en chino.
+> Intente consultar la documentación china siempre que sea posible.
+
+<Details>
+<Summary>Listado de Plugins</Summary>
+
+
+md
+Copiar código
```
Comment (sgkoishi) [inline README_ES.md:67]: `Copiar código`?

Comment (itsFrankV22) [reply README_ES.md:67]: It's because I was copying from the markdown website and by mistake I copied that

**Review by ACaiCat — APPROVED** [review]

File: README_ES.md (hunk 1 above)
Comment (ACaiCat) [reply README_ES.md:31]: 确实, （不过感觉问题不大

**Comment by ACaiCat** [comment]
Thanks @itsFrankV22!

**Pull request merged**